PHP/PHPhomework4.php

```diff
@@ -0,0 +1,17 @@
+<?php
+function findPrimeNumberBetween($n)
+{
+    for ($m = 2; $m <= $n; $m++) {
+        $x = 0;
+        $y = 1;
+        for ($i = 2; $i < $m; $i++) {
+            $x = $m % $i;
+            $y *= $x;
+        }
+        if($y !== 0) {
+            echo "$m"."<br>";
+        }
+    }
+}
+findPrimeNumberBetween();
```

Comment:
@toru828
I think you missed `$n`

Reply:
Where should I put $n?

Reply:
https://github.com/toru828/basic-homework/pull/1#discussion_r616282078
same to this comment